src/Header/Header.stories.tsx

```diff
@@ -0,0 +1,205 @@
+import React, { Meta, StoryObj } from '@storybook/react';
+import { within, userEvent } from '@storybook/testing-library';
+import { Header } from './Header';
+import { Button } from '../Button';
+import { desktopData, mobileData } from './data';
+import { Link } from '../Link';
+
+const meta: Meta<typeof Header> = {
+  title: 'Components/Header',
+  component: Header,
+};
+
+export default meta;
+type Story = StoryObj<typeof Header>;
+
+// Mock PointerEvent for play functions
+// Radix uses PointerEvent for all interactions
+// see: https://github.com/radix-ui/primitives/issues/1220
+class MockPointerEvent extends Event {
+  button: number;
+  ctrlKey: boolean;
+  pointerType: string;
+
+  constructor(type: string, props: PointerEventInit) {
+    super(type, props);
+    this.button = props.button || 0;
+    this.ctrlKey = props.ctrlKey || false;
+    this.pointerType = props.pointerType || 'mouse';
+  }
+}
+
+window.PointerEvent = MockPointerEvent as any;
+
+export const DesktopLight: Story = {
+  args: {
+    theme: 'light',
+    desktopActive: undefined,
+    desktopData: desktopData,
+    mobileData: mobileData,
+    desktopBreakpoint: 940,
+    desktopRight: (
+      <>
+        <Link>Sign in</Link>
+        <Button size="sm" variant="outline" color="blue">
+          Sign up
+        </Button>
+      </>
+    ),
+    mobileBottom: (
+      <div style={{ display: 'flex', flexDirection: 'column', gap: 12 }}>
+        <Button size="sm" variant="solid" color="blue">
+          Sign up
+        </Button>
+        <Button size="sm" variant="outline" color="blue">
+          Sign In
+        </Button>
+      </div>
+    ),
+  },
+  parameters: {
+    layout: 'fullscreen',
+    chromatic: { viewports: [940, 1024, 1280, 1536] },
+    viewport: {
+      defaultViewport: 'lg',
+    },
+  },
+};
+export const DesktopDark: Story = {
+  args: {
+    ...DesktopLight.args,
+    theme: 'dark',
+  },
+  parameters: {
+    ...DesktopLight.parameters,
+    backgrounds: { default: 'dark' },
+  },
+};
+
+export const DesktopLightActive: Story = {
+  args: {
+    ...DesktopLight.args,
+    desktopActive: 'Customers',
+  },
+  parameters: {
+    ...DesktopLight.parameters,
+  },
+};
+export const DesktopDarkActive: Story = {
+  args: {
+    ...DesktopLight.args,
+    theme: 'dark',
+    desktopActive: 'Customers',
+  },
+  parameters: {
+    ...DesktopLight.parameters,
+    backgrounds: { default: 'dark' },
+  },
+};
+
+export const DesktopLightOpen: Story = {
+  args: {
+    ...DesktopLight.args,
+  },
+  parameters: {
+    ...DesktopLight.parameters,
+  },
+  decorators: [(storyFn) => <div style={{ height: '800px' }}>{storyFn()}</div>],
+  play: async ({ canvasElement }) => {
```

Comment:
I tried adding a story to open one of the menus. It works locally but not in Chromatic. I'll look into it more tomorrow. Worst case, we might have to add a story just for the menu content.

Reply:
The solution I built yesterday was not working on Chromatic? What's the benefit of this new version?

Reply:
No. It wasn't opening the menus. But also, this is a proper test. It simulates what a user would do.